src/mme-app/msgHandlers/gtpMsgHandler.cpp

```diff
@@ -34,24 +34,20 @@ GtpMsgHandler* GtpMsgHandler::Instance()
 	return &msgHandler;
 }
 
-void GtpMsgHandler::handleGtpMessage_v(IpcEventMessage* eMsg)
+void GtpMsgHandler::handleGtpMessage_v(IpcEMsgUnqPtr eMsg)
 {
-    if (eMsg == NULL)
+    if (eMsg.get() == NULL)
         return;
 
     utils::MsgBuffer *msgBuf = eMsg->getMsgBuffer();
     if (msgBuf == NULL)
     {
         log_msg(LOG_INFO, "GTP Message Buffer is empty \n");
-
-        delete eMsg;
         return;
     }
     if (msgBuf->getLength() < sizeof(gtp_incoming_msg_data_t))
     {
         log_msg(LOG_INFO, "Not enough bytes in gtp message \n");
-
-        delete eMsg;
```

Comment:
we are getting rid of this since its unique pointer right ?

Reply:
Yes.